app/controllers/admin/admin_controller.rb

```diff
@@ -1,3 +1,19 @@
 class Admin::AdminController < ApplicationController
   layout "admin"
+
+  def admin_title(model=nil, keys = [])
+    return t('.index_title') unless model.present?
```

Comment:
I thought I understood "lazy lookup", but this confuses me.

What provides the context to instruct which `index_title` will be used when?